benchmarks/perf-tool/okpt/test/steps/steps.py

```diff
@@ -429,6 +429,118 @@ def bulk_transform_with_attributes(self, partition: np.ndarray, partition_attr,
         return actions
 
 
+class IngestNestedFieldStep(BaseIngestStep):
```

Comment:
we need to document this new step type usage in https://github.com/opensearch-project/k-NN/blob/feature/multi-vector/benchmarks/perf-tool/README.md#steps